harvard-lts/hdc3a-dlq-handler-service
# Service health check

**Service health check**
* * *
_Note: [PR #2](https://github.com/harvard-lts/hdc3a-dlq-handler-service/pull/2) should be reviewed and merged first._

**GitHub Issue**: 
**([link](https://app.zenhub.com/workspaces/hdc-62163f60fb08c60011fa4795/issues/harvard-lts/hdc/184))**

**Other Relevant Links**: 
**[PR #2](https://github.com/harvard-lts/hdc3a-dlq-handler-service/pull/2)**

# What does this Pull Request do?
Adds service health check with MQ connectivity checkers

# How should this be tested?

A description of what steps someone could take to:

* Make sure test .env variables are updated based on .env.example file
* Rebuild test environment: docker-compose -f docker-compose-test.yml build --no-cache
* Run the local test environment: `docker-compose -f docker-compose-test.yml up`
* Execute tests: `docker exec -it test_runner pytest test`

For a functional test:

* Make sure .env variables are updated based on .env.example file
* Rebuild dev environment: docker-compose -f docker-compose-dev.yml build --no-cache
* Run the local dev environment: `docker-compose -f docker-compose-dev.yml up`
* Access health check by GET https://localhost:10580/health

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests Yes
- integration tests Yes

# Interested parties
@jessjass @ives1227 @adaybujeda 

Keyword: resolved
